src/styles/components/_detailsBanner.scss

```diff
@@ -1,18 +1,30 @@
+$breakpoint-tablet: 820px;
+
 .details-banner {
   font-weight: 700;
```

Comment:
It looks like the font weight is `Regular` for the details banner title on the Figma

Reply:
I've also noticed that the banner title could also use more bottom spacing to match the Figma design